Tests/ScipioKitTests/DescriptionPackageTests.swift

```diff
@@ -74,10 +74,12 @@ final class DescriptionPackageTests: XCTestCase {
         XCTAssertEqual(package.name, "TestingPackage")
 
         XCTAssertEqual(
-            Set(try package.resolveBuildProducts().map(\.target.name)),
+            try package.resolveBuildProducts().map(\.target.name),
```

Comment:
By using `Array` instead of `Set`, this must be care about their orders. How about adding `sorted` instead?

Reply:
Isn't the idea also to check the order? `resolveBuildProducts` sorts all targets topologically and then reverses the order so that the build process can start from the leaf nodes. If I understand correctly, shouldn't the order also be preserved in the test?

```swift
[
                "Logging", // The leaf node
                "MyTarget", // Depends only on "Logging"
                "ExecutableTarget", // Depends only on "MyTarget"
                "MyPlugin", // Depends only on "ExecutableTarget", also the top package exposed via "products"
            ]
```

Reply:
oops,  I misunderstood, the order must be kept. It's great to add comments like your comment.

Reply:
Done by accepting your suggestion.